js/apps/admin-ui/src/components/users/UserDataTable.tsx

```diff
@@ -112,6 +115,36 @@ export function UserDataTable() {
   const [key, setKey] = useState(0);
   const refresh = () => setKey(key + 1);
 
+  const handleChange = async (
+    checked: boolean,
+    event: React.FormEvent<HTMLInputElement>,
+  ) => {
+    try {
+      await adminClient.users.update(
+        { id: event.currentTarget.id },
+        toUserRepresentation({
+          enabled: checked,
+        }),
+      );
+      refresh();
+    } catch (error) {
+      console.log(error);
+      refresh();
+    }
+  };
+
+  // eslint-disable-next-line react/no-unstable-nested-components
```

Comment:
This rule should not be violated, as it can create [performance problems and bugs](https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/no-unstable-nested-components.md#disallow-creating-unstable-components-inside-components-reactno-unstable-nested-components).